plugin.video.liveleak/addon.xml

```diff
@@ -19,6 +19,6 @@
         <license>GNU General Public License Version 3</license>
```

Comment:
Would be nice to change this to GPL-3.0-or-later (spdx identifier) in some future update